dist/images/ovnkube.sh

```diff
@@ -1890,6 +1894,11 @@ ovnkube-controller-with-node() {
     fi
   fi
 
+  ovnkube_node_lease_namespace_flag=
+  if [[ ${ovnkube_node_lease_namespace} != "" ]]; then
+    ovnkube_node_lease_namespace_flag="--ovnkube-node-lease-namespace=${ovnkube_node_lease_namespace}"
```

Comment:
where is this arg defined in go?

Reply:
This part was part of my initial PR. I just did not plan to make it configurable through the chart. https://github.com/Mellanox/ovn-kubernetes/blob/8463ec091a1509830ed99a9d09ad0aed29b63b02/go-controller/pkg/config/config.go#L1534

Reply:
awesome!